src/page/About.js

```diff
@@ -176,11 +176,11 @@ def main(): Unit & Impure =
     } |> println`}
                         </InlineEditor>
 
-                        <h2>Polymorphic Effects: Separating Pure and Impure Code</h2>
+                        <h2>Polymorphic Effects: Separating Pure and Effectful Code</h2>
 
                         <p>
                             A unique feature of Flix is its polymorphic effect system. The Flix type and effect system
-                            cleanly separates pure and impure code. If an expression is pure then it always
+                            cleanly separates pure and effectful code. If an expression is pure then it always
```

Comment:
I do not know, if it makes sense to just still write `impure`?

Reply:
I'm not even sure if it is correct to use effectful here? Is an `effectful function` the same as an `impure function` and a `functions with IO effect`?

Reply:
I think here its fine to say separate pure and impure code.

Reply:
an impure function and an effectful function is more or less the same.